analyzer/src/funTest/assets/projects/synthetic/maven-expected-output-app.yml

```diff
@@ -2,7 +2,9 @@
 project:
   id: "Maven:com.here.ort.maven:maven-app:1.0-SNAPSHOT"
   definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/maven/app/pom.xml"
-  declared_licenses: []
+  declared_licenses:
+  - "Apache License, Version 2.0"
```

Comment:
Why do we now have a new declared license here?

Reply:
To test support for declared license processing in Maven. Added an explanation to the commit message.